client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.controller.js

```diff
@@ -197,6 +201,8 @@ angular.module("managerApp")
                     serviceName: $stateParams.serviceName
                 }).$promise.then(function (maximumAvailableSimultaneousLines) {
                     self.options.maximumAvailableSimultaneousLines = maximumAvailableSimultaneousLines.maximum;
+                    self.numberLinesTitle = $translate.instant("between") + self.options.minimumAvailableSimultaneousLines + " " +
```

Comment:
see previous comment on translations file and use `$translate.instant` like this (it should work) :
```javascript
self.numberLinesTitle = $translate.instant("common_between_and", {
    between: self.options.minimumAvailableSimultaneousLines,
    and: self.options.maximumAvailableSimultaneousLines
});
```